lib/routes/api/metadata/processor.js

```diff
@@ -43,9 +43,6 @@ module.exports = function(url, options) {
       function done(json) {
         json = json || {};
         debug('parsed', json);
-        if (!json.title) {
-          return Promise.reject(new Error('empty'));
-        }
```

Comment:
It's only the 'title' field? Should the client be depending on a non-empty title?